source/lattice-construction.md

```diff
@@ -0,0 +1,158 @@
+(c:lattice-construct)=
+# Lattice Construction
+
+%---------------------------------------------------------------------------------------------------
+(s:lattice.construct)=
+## Constructing a Lattice
+
+A `Lattice` is essentially a set of branches. 
+Each branch is instantiated from a `BeamLine` which is called the a `root BeamLine`.
+Example:
+```{code} yaml
+Lattice:
+  - Branch: this_line    # Root beamline.
+  - Branch:
+      name: some_name
+      BeamLine: that_ring
+      periodic: True
+```
+In this example, `this_line` and `that_ring` are the names of the root BeamLines
+for the two `Branches` that will be created when the lattice is [expanded](#s:expansion).
+Branches created due to `Fork` elements also have root `BeamLines`. `Branches` specified
+in the `Lattice` structure are called `root branches` of the `lattice`. Non-root branches
+are those branches created due to `Fork` elements.
+
+A `Branch` has the optional components
+```{code} yaml
+name      # Optional String. Name of the branch. Default is the name of the root BeamLine.
+periodic  # Optional Bool. Are orbit and Twiss parameters periodic? 
+          #   Default is the setting of the root BeamLine.
+```
+The setting of `periodic` or `name` for a `Branch` overrides the setting of `periodic` or `name`
+set in the root `BeamLine`. See [](#s:beamline.components) for documentation of `periodic`.
+
+%---------------------------------------------------------------------------------------------------
+(s:branch.expand)=
+## Branch Expansion
+
+`Branch expansion` is the process, starting from the `root BeamLine`
+of a branch, of constructing the ordered list of lattice elements contained in that branch.
+
+The first lattice element of any root `BeamLine` line must be a `Beginning` element and the expanded
+branch line may only contain this one `Beginning` element. If a subline contains a `Beginning`
+element, this element must be dropped from the branch line.
+
+The last lattice element of a branch line be a `Marker` element.
```

Comment:
One or more words missing in this sentence?

Reply:
Added `must be`.